src/components/reference-book/page.cjsx

```diff
@@ -13,7 +13,7 @@ GetPositionMixin = require '../get-position-mixin'
 {ReferenceBookStore} = require '../../flux/reference-book'
 {ReferenceBookExerciseStore} = require '../../flux/reference-book-exercise'
 
-EXERCISE_MATCHER = new RegExp('#ost\/api\/ex\/(.*)')
+EXERCISE_MATCHER = new RegExp('\/exercises\/(.*)')
```

Comment:
Ah, from https://github.com/openstax/tagging-legend#exercises `.os-exercise .os-embed` should do the trick!